pyproject.toml

```diff
@@ -13,7 +13,7 @@ paramiko = "^2.7.1"
 spurplus = "^2.3.3"
 dataclasses-json = "^0.5.2"
 # portalocker for compatibility of Windows, trigger pywin32 installed
-portalocker = "^1.7.1" 
+portalocker = "^1.7.1"
 azure-identity = {version = "^1.4.0", allow-prereleases = true}
 azure-mgmt-resource = {version = "^15.0.0-beta.1", allow-prereleases = true}
 azure-mgmt-compute = {version = "^17.0.0-beta.1", allow-prereleases = true}
```

Comment:
Can you have a check if azure packages work without pre-release condition? I added it since there is a hard dependency, but it's in stable version already.

Reply:
I can, soon.